db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb

```diff
@@ -0,0 +1,6 @@
+class AddDateTypeToContentViewErratumFilterRules < ActiveRecord::Migration
+  def change
+    add_column :katello_content_view_erratum_filter_rules, :date_type, :string, :default => "updated", :null => false
+    Katello::ContentViewErratumFilterRule.update_all(:date_type => "issued")
```

Comment:
Or change this to: `update "UPDATE katello_content_view_erratum_filter_rules SET date_type = 'issued'"`.